p2p/net/swarm/swarm_dial.go

```diff
@@ -44,6 +44,9 @@ var (
 // add loop back in Dial(.)
 const dialAttempts = 1
 
+// dial at most 12 peers concurrently
+const concurrentPeerDials = 12
```

Comment:
thats really hard to tell at this point. The lock doesnt specify whats going to be dialed, just who.

How about dropping the per peer concurrency to 6 and raising the number of peers to 15? That puts us at a limit of 180 fd's (at least until we fix epoll)